src/Files.App/Views/SettingsPages/Tags.xaml.cs

```diff
@@ -66,6 +67,11 @@ private void CancelRenameTag_Click(object sender, RoutedEventArgs e)
 			CloseEdit();
 		}
 
+		private void CancelRemoveTag_Click(object sender, RoutedEventArgs e)
+		{
+			((Popup)((FlyoutPresenter)((StackPanel)((StackPanel)((Button)sender).Parent).Parent).Parent).Parent).IsOpen = false;
```

Comment:
Is there any other way😬 

`x:Bind`

Reply:
I may change it to `FindAscendant<Popup>()`

Reply:
Not sure if we can bind to anything. Flyouts have no `IsOpen` or similar properties

Reply:
I pushed a commit, guess that's the best we can achieve